lesson_7/ilas_lorena/decorators_3.py

```diff
@@ -0,0 +1,34 @@
+# Create a decorator called register that will update a list called print_registry with all the decorated functions
+# names.
+
+
+def register(fnc):
+    
+    def updater(args):
```

Comment:
This code can be further simplified; is the inner function really needed?
Justify the answer, whichever that might be

Reply:
I don't really need it because I'm just printing fnc.__name__  and nothing else. I will update the code accordingly.